generated/nidcpower/nidcpower/session.py

```diff
@@ -4869,11 +4886,27 @@ def measure_multiple(self):
 
         '''
         import collections
-        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
+        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance', 'channel'])
 
         voltage_measurements, current_measurements = self._measure_multiple()
 
-        return [Measurement(voltage=voltage_measurements[i], current=current_measurements[i], in_compliance=None) for i in range(self._parse_channel_count())]
+        with _NoChannel(session=self):
+            all_channels_in_session = self._get_channel_names(range(self.channel_count))
+
+        channel_names = _converters.expand_channel_string(
+            self._repeated_capability,
+            all_channels_in_session
+        )
```

Comment:
assert lenght of channel_names is the same as voltage_measurements and current_measurements

Reply:
I have added this assertion for `measure_multiple()` in my latest commit.